connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailRouteBuilder.java

```diff
@@ -303,4 +306,23 @@ private Endpoint setUpITEndpoint() throws Exception {
 
         return httpComponent.createEndpoint(String.format("https:%s", this.emailConnectorConfig.getItUserServiceURL()));
     }
+
+    /**
+     * Creates the endpoint for the BOP service. It makes Apache Camel trust
+     * BOP service's certificate.
+     * @return the created endpoint.
+     * @throws Exception if the endpoint could not be created due to an
+     * unexpected error.
+     */
+    protected Endpoint setUpBOPEndpoint() throws Exception {
+        final TrustManagersParameters trustManagersParameters = new TrustManagersParameters();
+        trustManagersParameters.setTrustManager(new BOPTrustManager());
+        final SSLContextParameters sslContextParameters = new SSLContextParameters();
+        sslContextParameters.setTrustManagers(trustManagersParameters);
+
+        final HttpComponent httpComponent = this.getCamelContext().getComponent("https", HttpComponent.class);
+        httpComponent.setSslContextParameters(sslContextParameters);
+
+        return httpComponent.createEndpoint(this.emailConnectorConfig.getBopURL());
```

Comment:
You could configure the certificates-related stuff at the endpoint level (see `ServiceNowRouteBuilder`) rather than changing everything at the `https` component level. Also, do we need the same things than the code below?
https://github.com/RedHatInsights/notifications-backend/blob/ba787fd66d00abae08f4325a2eff8501302bc677/connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowRouteBuilder.java#L60